packages/farcaster-js-hub-rest/src/hubRestApiClient.ts

```diff
@@ -168,7 +167,7 @@ export class HubRestAPIClient {
       mentionsPositions?: number[]
     },
     fid: number,
-    signer: NobleEd25519Signer,
+    signer: string,
```

Comment:
Nice, I agree with your idea to ask the dev for a privkey hex string instead of a typed NobleEd25519Signer object.

I think it will require that we make the parameter name more descriptive now, though, since we lost the documentation that was previously provided by the parameter's type annotation.

What about something like `signerPrivateKeyHex`?